<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Prueba</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Conectar_Bt">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar Conexión USB</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="medicion">
     <attribute name="title">
      <string>Medición</string>
     </attribute>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>561</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Bt_Abortar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>190</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Abortar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Bt_Capturar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>240</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Capturar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Bt_Exportar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>240</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Exportar</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tipoMedicion">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>191</width>
        <height>171</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Barrido Lineal</string>
       </attribute>
       <widget class="QPushButton" name="Bt_IniciarLineal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Iniciar Medición</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Num_VLineal">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <double>3.300000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Tensión máxima [V]</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Frecuencia [Hz]</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Num_HzLineal">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Barrido Cíclico</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Tensión máxima [V]</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Num_VCiclico">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>10</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <double>3.300000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Num_HzCiclico">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QPushButton" name="Bt_IniciarCiclico">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>100</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Iniciar Medición</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Frecuencia [Hz]</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="Num_CicCiclico">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Cant. de Ciclos</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Desconectar_Bt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Terminar Conexión USB</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNueva_Medicion"/>
    <addaction name="actionAbrir_Medicion"/>
    <addaction name="actionOpciones"/>
   </widget>
   <widget class="QMenu" name="menuBarrido_Lineal">
    <property name="title">
     <string>Barrido Lineal</string>
    </property>
    <addaction name="actionMedir"/>
    <addaction name="actionConfiguracion"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuBarrido_Lineal"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNueva_Medicion">
   <property name="text">
    <string>Nueva Medición</string>
   </property>
  </action>
  <action name="actionAbrir_Medicion">
   <property name="text">
    <string>Abrir Medición</string>
   </property>
  </action>
  <action name="actionOpciones">
   <property name="text">
    <string>Opciones</string>
   </property>
  </action>
  <action name="actionMedir">
   <property name="text">
    <string>Medir</string>
   </property>
  </action>
  <action name="actionConfiguracion">
   <property name="text">
    <string>Configuración</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
